cirq/experiments/xeb_fitting_test.py

```diff
@@ -83,6 +94,49 @@ def test_benchmark_2q_xeb_fidelities_parallel():
     n_pairs = sum(len(c.pairs) for c in combs)
     assert len(fid_df) == len(cycle_depths) * n_pairs
 
+    fit_df = fit_exponential_decays(fid_df)
+    for _, row in fit_df.iterrows():
+        assert list(row['cycle_depths']) == list(cycle_depths)
+        assert len(row['fidelities']) == len(cycle_depths)
+
+
+def test_benchmark_2q_xeb_fidelities_vectorized():
```

Comment:
Would it be possible to have a test with hard-coded expected values of all the intermediate variables instead of realying on two different implementations of the same logic?

Reply:
that seems more fragile, no? having two independent implementations is the best possible unit test!

Reply:
How do you test if the other implementation doesn't have a bug then?

Reply:
there are other tests that test the whole function, which the previous and current one is passing. This is an additional test to make sure the two implementations are identical.